app/src/main/java/org/supla/android/lib/SuplaClient.java

```diff
@@ -891,6 +899,11 @@ private void onRegistered(SuplaRegisterResult registerResult) {
     SuplaClientMsg msg = new SuplaClientMsg(this, SuplaClientMsg.onRegistered);
     msg.setRegisterResult(registerResult);
     sendMessage(msg);
+
+    String token = preferences.getFcmToken();
+    if (token != null) {
```

Comment:
If the token is null, send "". Then the server will not send notifications if it previously had a registered token.